src/services/authService.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
```

Comment:
you can create a custom axios instance. So, you can configurate the `API BASE URL`, `TIME OUT` and `HEADERS`  
https://www.npmjs.com/package/axios#axioscreateconfig

then just do
`axios.post('users/sessions', { email, password });`